src/components/tabs/buy-tab-content/buy.process.tsx

```diff
@@ -159,27 +163,42 @@ interface PaymentInformation {
 
 interface PaymentInformationContentProps {
   info: PaymentInformation;
+  onFinish: () => void;
 }
 
-function PaymentInformationContent({ info }: PaymentInformationContentProps): JSX.Element {
+function PaymentInformationContent({ info, onFinish }: PaymentInformationContentProps): JSX.Element {
+  const { copy } = useClipboard();
   return (
     <>
       <h2>Payment Information</h2>
       <p>
         Please transfer the purchase amount using this information via your banking application. The purpose of payment
         is important!
       </p>
-      <p>IBAN: {info.iban}</p>
-      <p>BIC: {info.bic}</p>
-      <p>Purpose {info.purpose}</p>
-      <p>Recipient: {info.recipient}</p>
-      <p>Fee: {info.fee}</p>
-      <div className="border border-dfxGray-500 rounded-md"></div>
-      <StyledButton
-        label="Click once your bank Transfer is completed."
-        onClick={() => console.log('TODO ask what should happen after this button was clicked')}
-        caps={false}
-      />
+      <StyledDataTable darkTheme={false} alignContent={AlignContent.RIGHT} showBorder>
+        <StyledDataTableRow label="IBAN">
+          {info.iban}
+          <StyledIconButton icon={IconVariant.COPY} onClick={() => copy(info.iban)} />
+        </StyledDataTableRow>
+        <StyledDataTableRow label="BIC">
+          {info.bic}
+          <StyledIconButton icon={IconVariant.COPY} onClick={() => copy(info.bic)} />
+        </StyledDataTableRow>
+        <StyledDataTableRow label="Purpose of payment">
+          {info.purpose}
+          <StyledIconButton icon={IconVariant.COPY} onClick={() => copy(info.purpose)} />
+        </StyledDataTableRow>
+      </StyledDataTable>
+      <StyledDataTable label="Recipient" darkTheme={false} showBorder>
+        <StyledDataTableRow>DFX AG, Bahnhofstrasse 7, 6300 Zug, Schweiz</StyledDataTableRow>
+      </StyledDataTable>
+      <StyledDataTable darkTheme={false} alignContent={AlignContent.BETWEEN}>
+        <StyledDataTableRow discreet>
+          <p>DFX-Fee</p>
+          <p>{info.fee}</p>
+        </StyledDataTableRow>
+      </StyledDataTable>
+      <StyledButton label="Click once your bank Transfer is completed." onClick={onFinish} caps={false} />
```

Comment:
Somewhat weird to have the button in the payment info component? 🤔

Reply:
was on brink to moving it out. will do